libraries/standard/mqtt/include/mqtt_state.h

```diff
@@ -118,4 +123,13 @@ uint16_t MQTT_StateSelect( const MQTTContext_t * pMqttContext,
                            MQTTPublishState_t searchState,
                            MQTTStateCursor_t * pCursor );
 
+/**
+ * @brief Error code to string conversion for state engine.
```

Comment:
Is it error code or publish state i.e. Publish State to string conversion for state engine.